test/e2e/view-add-reduction.js

```diff
@@ -1,17 +1,18 @@
 const expect = require('chai').expect
 const authenticationHerlp = require('../helpers/routes/authentication-helper')
 const dataHelper = require('../helpers/data/aggregated-data-helper')
+const workloadTypes = require('../../app/constants/workload-type')
 
 var offenderManagerId
 var offenderManagerUrl
 
-describe('View adding a new reduction', () => {
+describe('View adding a new reduction - court reports', () => {
   before(function () {
     authenticationHerlp.login(authenticationHerlp.users.Manager)
     return dataHelper.getAnyExistingWorkloadOwnerId()
         .then(function (results) {
           offenderManagerId = results
-          offenderManagerUrl = '/offender-manager/' + offenderManagerId + '/add-reduction'
+          offenderManagerUrl = '/' + workloadTypes.PROBATION + '/offender-manager/' + offenderManagerId + '/add-reduction'
```

Comment:
description says court reports yet the URL is probation?

Reply:
good spot,  removed court-report